include/ruby/backward/2/stdalign.h

```diff
@@ -20,11 +20,11 @@
  *             extension libraries. They could be written in C++98.
  * @brief      Defines #RUBY_ALIGNAS / #RUBY_ALIGNOF
```

Comment:
Should be `#RBIMPL_ALIGNAS / #RBIMPL_ALIGNOF`

Reply:
This particular header file does define `#RUBY_ALIGNAS` / `#RUBY_ALIGNOF`, which have been around for years.  We retain these macros for backward compatibility.

Reply:
I misread, indeed, all fine.